python/datashaper/datashaper/utils/parallelization/types.py

```diff
@@ -0,0 +1,10 @@
+"""Execution utility typings."""
+
+from enum import Enum
+
+
+class AsyncType(str, Enum):
```

Comment:
I've been wary of "*Type" lately in python, and had to change some things in DataShaper to avoid conflicts (e.g., we had params like `type: ComparisonType`), which I've changed to somewhat equivalents like "*Strategy" (so now `strategy: ComparisonStrategy` for instance) to avoid weird potential built-in issues. Anyway, maybe worth considering on the name and subsequent usage here

Reply:
That's fair, using another word here can probably let us be more accurate with some of these names. I've renamed a couple of these enums.